models/evc.py

```diff
@@ -751,7 +751,6 @@ def remove_current_flows(
             current_path.make_vlans_available(self._controller)
         except KytosTagError as err:
             log.error(f"Error removing {self} current_path: {err}")
-            return old_path_dict
```

Comment:
This line is present in `2023.2.8`, the other issues are not. This line will probably never triggered since `make_vlans_available` always sends `"vlan"` but it could be confusing for developers.